addons/web_editor/static/src/js/wysiwyg/widgets/link_tools.js

```diff
@@ -425,6 +444,18 @@ export class LinkTools extends Link {
     _onClickCheckbox(ev) {
         const $target = $(ev.target);
         $target.closest('we-button.o_we_checkbox_wrapper').toggleClass('active');
+        if ($target[0].getAttribute("name") === "direct_download") {
+            const el = $target.closest("we-button.o_we_checkbox_wrapper")[0];
```

Comment:
can probably just use ev.target for everthing to avoid referencing jQuery, except if you see a difference in behaviour?
```suggestion
            const el = ev.target.closest("we-button.o_we_checkbox_wrapper");
```